### PR TITLE
Fix decimalPrecision for min and maxValue in class

### LINKED ISSUE #14849: [Bug]: In min. Value or max. Value of Number field of class can't enter less 0.01
### Expected behavior

In class edit in number field I can enter in "min. Value:" (or "max. Value:") less than 0.01

### Actual behavior

Then I enter something less than 0.01 it is chenging to 0 (in min. Value or max. Value number field of class).

### Steps to reproduce

In min. Value or max. Value number field of class try to enter 0.001. It will change to 0.

### PR DESCRIPTION
**Pull request by MasterOfDeath**
## Changes in this pull request  
Resolves #14849

### CODE REVIEW

**Comment by lukmzig** [comment]
@MasterOfDeath thank you very much for the contribution. 
As far as I can see, this solution would work only if you save and reload the class definition. Maybe it would be possible to add a change listener to the _decimaPrecision_ field and change the decimal precision of the _minValue_ and _MaxValue_ dynamically.

Could you please have a look? Thank you!

**Comment by MasterOfDeath** [comment]
> @MasterOfDeath thank you very much for the contribution. As far as I can see, this solution would work only if you save and reload the class definition. Maybe it would be possible to add a change listener to the _decimaPrecision_ field and change the decimal precision of the _minValue_ and _MaxValue_ dynamically.
> 
> Could you please have a look? Thank you!

ok, I'll do it

**Comment by MasterOfDeath** [comment]
@lukmzig Done

File: bundles/AdminBundle/Resources/public/js/pimcore/object/classes/data/numeric.js [hunk 1]
```diff
@@ -111,7 +111,22 @@ pimcore.object.classes.data.numeric = Class.create(pimcore.object.classes.data.d
...
+
+                        if (minValueComponent) {
+                            minValueComponent.decimalPrecision = value;
+                        }
+
+                        if (maxValueComponent) {
+                            maxValueComponent.decimalPrecision = value;
+                        }
+                    }.bind(this)
+                }
```
Comment (lukmzig) [inline bundles/AdminBundle/Resources/public/js/pimcore/object/classes/data/numeric.js]:
```suggestion
                listeners: {
                    change: function (el, value) {
                        let minValueComponent = this.specificPanel.getComponent('minValue');
                        let maxValueComponent = this.specificPanel.getComponent('maxValue');

                        if (minValueComponent) {
                            minValueComponent.decimalPrecision = value;
                        }

                        if (maxValueComponent) {
                            maxValueComponent.decimalPrecision = value;
                        }
                    }.bind(this)
                }
```

@MasterOfDeath you can pass the value into the listener directly. Can you please adapt that?

Comment (MasterOfDeath) [reply bundles/AdminBundle/Resources/public/js/pimcore/object/classes/data/numeric.js]: @lukmzig Done it

Comment (lukmzig) [reply bundles/AdminBundle/Resources/public/js/pimcore/object/classes/data/numeric.js]: Thank you

**Pull request merged**